pty/ptytest/ptytest.go

```diff
@@ -72,11 +72,29 @@ func create(t *testing.T, ptty pty.PTY, name string) *PTY {
 		ctx, cancel := context.WithTimeout(context.Background(), testutil.WaitShort)
 		defer cancel()
 
-		_ = out.Close()
-		_ = ptty.Close()
+		errC := make(chan error, 2)
```

Comment:
nit: should `errC` be closed?

Reply:
Nope, we're waiting for `copyDone`, closing `errC` would exit cleanup too early.